bestpy/best.py

```diff
@@ -0,0 +1,15 @@
+from .answers import answers
+
+import random
+
+
+class Best:
+    def __getattribute__(self, name):
+        # Return random value in list
+        if isinstance(answers[name], list):
+            return random.choice(answers[name])
+        # Call function
+        elif type(answers[name]).__name__ == "function":
+            return answers[name]()
+        else:
+            return answers[name]
```

Comment:
```suggestion
    def __getattribute__(self, name):
        answer = answers[name]
        if callable(answer):
            answer = answer()
            
        if isinstance(answer, list):
            return random.choice(answer)
        else:
            return answer
```
Sometimes, the functions may return lists. We should expect the return values to act the same way that we expect the answers to act, if that makes sense. Calling it — if it's a function — could help provide the return value and let us continue parsing it as usual. This currently also looks slightly messy in my opinion.